packages/application-shell/src/components/navbar/use-navbar-state-manager.ts

```diff
@@ -256,6 +266,18 @@ const useNavbarStateManager = (props: HookProps) => {
     [state.activeItemIndex]
   );
 
+  const handleMouseMove = useCallback(
```

Comment:
nit: just noticed that we have two functions named `handleMouseMove` that do different things. Could we rename this one to be more descriptive?